README.md

```diff
@@ -15,24 +15,45 @@ Access to both documents is available to members of the
 [kubernetes-sig-multicluster google
 group](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster).
 
+# Concepts
+
 <p align="center"><img src="docs/images/concepts.png" width="711"></p>
 
-# Concepts
+Federation is configured with two types of information:
+
+- **Type configuration** declares which API types federation should handle
+- **Cluster configuration** declares which clusters federation should target
+
+**Propagation** refers to the mechanism that distributes resources to federated
+clusters.
 
-The following abstractions support the propagation of a logical
-federated type:
+Type configuration has three fundamental concepts:
 
-- Template: defines the representation of the resource common across clusters
-- Placement: defines which clusters the resource is intended to appear in
-- Override: optionally defines per-cluster field-level variation to apply to the template
+- **Template types** define the representation of a resource common across clusters
```

Comment:
How about remote `types` but only keep `Template` to keep consistent with diagram?

Ditto for others.